api/core/rag/extractor/excel_extractor.py

```diff
@@ -27,10 +28,37 @@ def __init__(
         self._autodetect_encoding = autodetect_encoding
 
     def extract(self) -> list[Document]:
+        """ parse excel file"""
+        if self._file_path.endswith('.xls'):
+            return self._extract4xls()
+        elif self._file_path.endswith('.xlsx'):
+            return self._extract4xlsx()
+
+    def _extract4xls(self) -> list[Document]:
+        wb = xlrd.open_workbook(filename=self._file_path)
+        row_header = None
```

Comment:
every sheet needs to initialize  row_header， maybe move this to line 43 after `for sheet in wb.sheets():`

Reply:
I'm sorry for not thinking of this part, he did have such logic, I will fix it